index.js

```diff
@@ -120,7 +158,14 @@ const resolveResponse = (response, options) => {
 
   const allEntries = [...responseClone.items, ...allIncludes]
 
-  const entityMap = new Map(allEntries.map((entity) => [makeLookupKey(entity.sys), entity]))
+  const entityMap = new Map(
+    allEntries.reduce((acc, entity) => {
+      const keys = makeEntityMapKeys(entity.sys)
+      acc.push([keys[0], entity])
+      acc.push([keys[1], entity])
+      return acc
```

Comment:
`makeEntityMapKeys` can return an array with only 1 value, making the key on line 165 `undefined`. 
```suggestion
      const entries = makeEntityMapKeys(entity.sys).map((key) => [key, entity]);
      acc.push(...entries)
      return acc
```